README.org

```diff
@@ -13,150 +13,75 @@ As Python 3 matures and majority python packages support Python 3, the scientifi
-    | signal peptide          | ongoing   | SignalP                                          |
-    | transmembrane proteins  | ongoing   | TMHMM                                            |
-
-* Databases Supported
-
-    | Databases | Supported |
-    |-----------+-----------|
-    | TIGRFAM   | yes       |
-    | UniRef    | yes       |
-    | Rfam      | ongoing   |
-
-* Getting help
-
-To get help with micronota, you should use the [[https://biostars.org/t/micronota][micronota tag]] on Biostars. The developers regularly monitor the =micronota= tag on Biostars.
-
-
-* Developing
-
-If you're interested in getting involved in micronota development, see [[https://github.com/biocore/micronota/blob/master/CONTRIBUTING.md][CONTRIBUTING.md]].
-
-See the list of [[https://github.com/biocore/micronota/graphs/contributors][micronota's contributors]].
-
+* Prepare Databases
+  You need to download database files:
+    | Databases | Supported | Download URL |
+    |-----------+-----------+--------------|
+    | UniRef    | yes       |              |
+    | Rfam      | yes       |              |
+    | ResFam    | ongoing   |              |
+
+* Usage
+** You can see the command supported by micronota with:
+   #+BEGIN_SRC sh
+     micronota -h
+   #+END_SRC
+** To run the default annotation pipeline:
+   #+BEGIN_SRC sh
+     micronota annotate -i <input.fna> -o <output-dir> --out-fmt genbank --kingdom bacteria
+   #+END_SRC
+** Customize the annotaton:
+   You can set up what annotation to run with what parameters by providing a config file when running annotation:
+   #+BEGIN_SRC sh
+     micronota annotate -i <input.fna> -o <output-dir> --out-fmt genbank --kingdom bacteria --config <your-config>
+   #+END_SRC
+
+   You can modify the [[https://github.com/biocore/micronota/blob/master/micronota/bacteria.yaml][default config file]] to create your own config file.
+
+* Sequence Features to Annotate
+
+    | Features                                 | Supported | Tools                                            |
+    |------------------------------------------+-----------+--------------------------------------------------|
+    | coding gene                              | yes       | Prodigal                                         |
+    | tRNA                                     | yes       | Aragorn                                          |
+    | ncRNA                                    | yes       | Infernal + Rfam                                  |
+    | CRISPR                                   | yes       | MinCED                                           |
+    | rho-independent transcription terminator | yes       | transtermhp                                      |
+    | tandem repeat                            | yes       | tandem repeat finder (trf)                       |
+    | ribosomal binding sites                  | ongoing   | RBSFinder                                        |
```

Comment:
maybe link "ongoing" with the corresponding GitHub issue to help track the progress?